flink-table/flink-table-planner/src/test/java/org/apache/flink/table/api/QueryOperationTestPrograms.java

```diff
@@ -740,4 +740,89 @@ private static Instant dayOfSeconds(int second) {
                             },
                             "MySink")
                     .build();
+
+    static final TableTestProgram OVER_WINDOW_RANGE =
```

Comment:
Should we also test the `UNBOUNDED PRECEDING` case here?
It is handled differently than the bounded preceding window case.

Reply:
+ we should also test `OVER` without `PARTITION BY` (and possibly without `ORDER BY`)